src/resources/parser/json-model.js

```diff
@@ -333,6 +406,14 @@ pc.extend(pc, function () {
                     meshInstance.skinInstance = skinInstances[skinIndex];
                 }
 
+                if (mesh.morph) {
+                    var morphIndex = morphs.indexOf(mesh.morph);
+                    if (morphIndex === -1) {
+                        throw new Error('Mesh\'s morph does not appear in morph array.');
```

Comment:
This is basically a copy-pasted code from skinning parser. Should change both then.